PhysicsTools/PatUtils/python/tools/muonRecoMitigation.py

```diff
@@ -0,0 +1,84 @@
+import FWCore.ParameterSet.Config as cms
+
+from FWCore.GuiBrowsers.ConfigToolBase import *
+
+def muonRecoMitigation(process,
+                       pfCandCollection,
+                       runOnMiniAOD,
+                       selection="",
+                       muonCollection="",
+                       cleanCollName="cleanMuonsPFCandidates",
+                       cleaningScheme="all",
+                       postfix=""):
+
+    sequence=cms.Sequence()    
+
+    if selection=="":
+        if runOnMiniAOD:
+            from RecoMET.METFilters.badGlobalMuonTaggersMiniAOD_cff import badGlobalMuonTagger, cloneGlobalMuonTagger
+        else:
+            from RecoMET.METFilters.badGlobalMuonTaggersAOD_cff import badGlobalMuonTagger, cloneGlobalMuonTagger
```

Comment:
To my understanding, `muonRecoMitigiation` is not actually used in the re-miniAOD setup.
I leave it to @mmarionncern to fix it for the purpose of MET recipes on existing (Mini)AOD.